atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfigsTest.java

```diff
@@ -115,8 +116,8 @@ public void canDeserializeRuntimeConfig() throws IOException {
 
     @Test
     public void canParseRuntimeDeprecatedConfigType() throws IOException {
-        CassandraKeyValueServiceRuntimeConfig config =
-                AtlasDbConfigs.OBJECT_MAPPER.readValue("type: CassandraKeyValueServiceRuntimeConfig",
-                        CassandraKeyValueServiceRuntimeConfig.class);
+        KeyValueServiceRuntimeConfig config =
```

Comment:
I think you want to still use the old form for this test since that's the deprecated version.

I would also change all the tests to deserialise to `KeyValueServiceRuntimeConfig.class` instead of the semi-concrete `CassandraKeyValueServiceRuntimeConfig` test to ensure we're doing the right thing everywhere